src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -401,6 +438,31 @@ def register_submission(self, submission: Submission, options: dict) -> dict | N
         if submission.has_registrator:
             result["medewerker_rol"] = medewerker_rol
 
+        # Register submission to Objects API if configured
+        if (
+            options.get("objecttype")
+            and options.get("objecttype_version")
+            and options.get("content_json")
+        ):
+            result["objects_api"] = self.register_submission_to_objects_api(
```

Comment:
I'd wrap this entire call into `execute_unless_result_exists` so you can avoid doing a bunch of work like templating out the JSON again if it's already registered.